### ISSUE TITLE
Conditional dips

### ISSUE THREAD

**Issue by eubrunosilva** [MiSTer-devel/Main_MiSTer#314]
Hi

this kind of dips I already found in more games... and maybe in the future we will find out in more and more

```
		<dipswitch name="Coinage" tag="IN2" mask="3">
			<condition tag="IN2" mask="16" relation="eq" value="0"/>
			<diplocation name="SW" number="1"/>
			<diplocation name="SW" number="2"/>
			<dipvalue name="2 Coins per Player" value="2"/>
			<dipvalue name="2 Coins/1 or 2 Players" value="3"/>
			<dipvalue name="1 Coin per Player" value="0" default="yes"/>
			<dipvalue name="1 Coin/1 or 2 Players" value="1"/>
		</dipswitch>
		<dipswitch name="Game Time" tag="IN2" mask="12">
			<condition tag="IN2" mask="16" relation="eq" value="0"/>
			<diplocation name="SW" number="3"/>
			<diplocation name="SW" number="4"/>
			<dipvalue name="60 seconds" value="0"/>
			<dipvalue name="70 seconds" value="4" default="yes"/>
			<dipvalue name="80 seconds" value="8"/>
			<dipvalue name="90 seconds" value="12"/>
		</dipswitch>
		<dipswitch name="Service Mode" tag="IN2" mask="16">
			<diplocation name="SW" number="5"/>
			<dipvalue name="Off" value="0" default="yes"/>
			<dipvalue name="On" value="16"/>
		</dipswitch>
```

in conditional dips this is one example but i can get more examples if needed

**Comment by sorgelig** [comment]
I'm not familiar with mame xml format. 
Please rephrase your question with details where i can see the problem.
Basically, conditional DIPs can be handled by core which already should know this condition. Core also may re-arrange the DIPs to convenient order (it's about previous issue with non-sequential DIP positions)
Core is not required to follow exact dip locations, so it can introduce its own DIP config which will be arranged by core to original arcade positions.

**Comment by jotego** [comment]
I don't understand this request well either.

**Comment by thehughhefner** [comment]
My suggestion is that this ticket be closed until @eubrunosilva comes back and elaborates further the request

**Comment by birdybro** [comment]
There is an example of this in Tropical Angel, where there is a Coin Mode DIP switch, the game would either use 4 bits to switch the behavior of using one coin slot for the number of coins to credits, or if the Coin Mode DIP switch was turned on it would make it so a dual Coin Slot configuration could have each coin slot configurable for different values. Really bizarre and I went with asserting the coin mode to the single-slot option.

I don't doubt there are more examples of this sort of thing. Below is the arcade operator's page from an owner of a Tropical Angel board and I confirmed the coin mode dip does change the behavior of dips 5-8 in the core:

![image](https://user-images.githubusercontent.com/16388068/202005881-e9287d5d-23d9-431b-8879-fba659c6c9ab.png)
